kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/document/datadictionary/PurchaseOrderAmendmentDocument.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" 
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xmlns:p="http://www.springframework.org/schema/p" 
+xmlns:dd="http://rice.kuali.org/dd" 
+xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd         http://rice.kuali.org/dd         http://rice.kuali.org/dd/dd.xsd">
+
+   <!-- workflow attributes for routing -->
+    <bean id="PurchaseOrderAmendmentDocument-workflowAttributes" parent="PurchaseOrderAmendmentDocument-workflowAttributes-parentBean">  
+    	<property name="routingTypeDefinitions">
+    		<map>
+    		    <!-- no qualifiers for HasNewUnorderedItems -->
+    			<entry key="Account" value-ref="RoutingType-PurchasingAccountsPayableDocument-Account"/>
+    			<entry key="AccountingOrganizationHierarchy" value-ref="RoutingType-PurchasingAccountsPayableDocument-AccountingOrganizationHierarchy"/>
```

Comment:
AccountingOrganizationHierarchy routing to be added in a later mod, so please remove for now.